### DISCUSSION TITLE
Arduino IDE compile error for an esp32-s3 mini

### DISCUSSION THREAD

**Discussion by AlainF-ing** [espressif/arduino-esp32#8980] [Q&A]
Hi I have used ESP32 core 2.0.14 to compile the FSBrowser.ino example and it works.
Using core 2.0.14 I was not able to load the SPIFFS on the ESP32-s3
Now on dev branch using 3.0.0 alpha3 I was able to load the SPIFFS on the ESP32-s3 mini
If I compile the FSBrowser example I receive many errors about a missing file 
The message give path and
 .............. liblto_plugin.so: cannot open shared object file: No such file or directory

I run on Raspberry Pi4 
Bullseye 32 bits latest update
Arduino IDE 1.8.19 
I have tried both ESP32 core 2.0.14 (stable) and 3.0.0 alpha3
connected on ESP32-s3 mini 

Arduino config 
CDC enabled
Default partition scheme 4Mb with SPIFFS (1.2Mb app 1.5 Mb SPIFFS)

[ESP32-s3compileErrors.txt](https://github.com/espressif/arduino-esp32/files/13629730/ESP32-s3compileErrors.txt)

Thanks for your help

**Comment by me-no-dev** [comment]
This looks like an issue with your OS and/or the toolchain for the new versions. 2.0.14 uses an older toolchain. Any chance you can try this on the 64bit flavor?

**Reply by AlainF-ing** [reply]
Ok I will try on PI OS version 64 bit and get back with results.

**Reply by AlainF-ing** [reply]
On my Raspberry Pi 4 
I have installed PI OS Bullseye legacy 64 bit with all updates 
Arduino 1.8.19 
Support for ESP32 core  3.0.0.alpha3
The compile work ok (no warning, no error)
The program loaded OK on esp32-s3 but the program does not seems to work correctly.
So I am back on regular program troubleshooting.
Thanks for your help

**Reply by me-no-dev** [reply]
Glad it worked!
